src/components/viz-pane/renderer.tsx

```diff
@@ -57,6 +103,24 @@ export default class VizPane extends React.PureComponent<Props> {
       this.props.showLogs(true);
     }
   }
+
+  public getComponent() {
```

Comment:
Which component?

Reply:
I just moved the position of the function , it is used to get the data-viewer,signal-viewer and the error based on which is selected

Reply:
I think it needs a qualifier in the name,

Reply:
What should be the name?

Reply:
You decide

Reply:
Something like getVizPane Component will do

Reply:
It's not a visualization pane, right? It's a `getContextViewer` or something. Describe in the comment that it returns the debug pane, data or signal viewer.

Reply:
Should I name it getContextViewer and add a comment explaining what it returns?

Reply:
👍